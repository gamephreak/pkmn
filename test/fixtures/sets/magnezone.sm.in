Maggy (Magnezone) @ No Item  
Trait: Sturdy  
Level: 50  
Shiny: Yes  
Happiness: 0  
EVs: 85 HP / 85 Atk / 85 Def / 85 SpA / 85 SpD / 85 Spe  
Serious nature  
IVs: 1 HP / 29 Def / 28 SpA / 26 SpD / 26 Spe  
~ Thunderbolt  
~
~ Flash Cannon  
~ Hidden Power [Flying]  
~ Volt Switch  
