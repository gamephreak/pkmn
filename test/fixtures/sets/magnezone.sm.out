Maggy (Magnezone) @ Choice Specs  
Ability: Magnet Pull  
Level: 50  
Shiny: Yes  
Happiness: 0  
EVs: 85 HP / 85 Atk / 85 Def / 85 SpA / 85 SpD / 85 Spe  
Serious Nature  
IVs: 1 HP / 30 Atk / 29 Def / 28 SpA / 26 SpD / 26 Spe  
- Thunderbolt  
- Flash Cannon  
- Hidden Power [Fire]  
- Volt Switch  

